resources/views/pages-legacy/userInfo.blade.php

```diff
@@ -59,6 +59,7 @@
 $totalSoftcoreAchievements = $progressionCounts['totalSoftcoreAchievements'];
 
 $userCompletedGamesList = getUsersCompletedGamesAndMax($userPage, limit: 200);
+$userIncompleteGamesList = getUsersCompletedGamesAndMax($userPage, limit: 200, isExcludingCompleted: true);
```

Comment:
Possible optimization:

```
if (count($userCompletedGamesList) < 200):
    $userIncompleteGamesList = $userCompletedGamesList->filter(...)
```

Though the benefit of avoiding the second query is minimal if there's a limited number of entries already.

Reply:
Good idea. Tested locally, but doesn't seem to make much/any of a dent in page load performance. Given that, simplicity may be desirable here - I'll hang on to the current implementation unless this comes back to bite us after a deploy.